contracts/ethregistrar/ETHRegistrarController.sol

```diff
@@ -182,14 +190,11 @@ contract ETHRegistrarController is Ownable, IETHRegistrarController {
             _setReverseRecord(name, resolver, msg.sender);
         }
 
-        emit NameRegistered(
-            name,
-            label,
-            owner,
-            msg.value - premiumCost,
-            premiumCost,
-            expires
-        );
+        emit NameRegistered(name, label, owner, cost[0], cost[1], expires);
+
+        if (msg.value > (cost[0] + cost[1])) {
+            payable(msg.sender).transfer(msg.value - (cost[0] + cost[1]));
```

Comment:
Is there a test for this logic? (I couldn't find it).

Reply:
I believe the logic where it tests the balance before and after in the contract tests this. Did you have a suggestion on how to test it?